lib/pltr/v2/selectors/ui.js

```diff
@@ -132,6 +136,17 @@ export const isCardDialogVisibleSelector = createSelector(cardDialogSelector, (c
   return cardDialog?.isOpen
 })
 
+export const bookDialogBookNumberSelector = createSelector(bookDialogSelector, (bookDialog) => {
```

Comment:
This selector should be able to compute what the book number is based on whether there is a `bookId` (i.e. the book dialog is open, and how many books there are already.